cmd/crc/cmd/root.go

```diff
@@ -90,17 +91,32 @@ func runRoot() {
 	fmt.Println("No command given")
 }
 
+const (
+	defaultExitCode         = 1
```

Comment:
This is an error code, when I read `defaultExitCode`, I assume it's going to be 0.
`generic/defaultErrorExitCode` maybe?

Reply:
done